notifyme-sdk-backend/notifyme-sdk-backend-ws/src/main/java/ch/ubique/notifyme/sdk/backend/ws/controller/NotifyMeControllerV3.java

```diff
@@ -226,21 +226,30 @@ private ProblematicEvent mapTraceKeyToProblematicEvent(TraceKey t) {
       consumes = {"application/x-protobuf", "application/protobuf"})
```

Comment:
https://github.com/notifyme-app/notifyme-sdk-backend/pull/12/files#diff-3831def3db4a6233be7fc0827a926221afb47b5e530e79a84b859067c8caa942R191 this request must be protected or removed for now, otherwise any traceKeys can be uploaded.